app/lib/back-helpers.js

```diff
@@ -65,7 +65,7 @@ const addBackNavigation = (request, markAsMainReturnPoint = false) => {
   setInSession(request, lastHashParam, newHashParam)
 
   return {
-    backLink: backLinkUrl,
+    backLink: backLinkUrl.replace('&clear=true', ''),
```

Comment:
This may impact other calls. Can you correct this at source rather than in thins common method?

Reply:
yeah, it was throwing up errors - didn't realise it was in the same place, have removed and added to the model.